packages/appkit/src/universal-adapter/client.ts

```diff
@@ -258,6 +264,8 @@ export class UniversalAdapterClient {
 
       writeContract: async () => await Promise.resolve('0x'),
 
+      grantPermissions: async () => await Promise.resolve('0x'),
```

Comment:
Why wagmi we rely on Universal Provider to execute the request and here we don't use it?

Reply:
I wasn't sure as for other methods no implementation, following is the comment on the file. 
`// -- Transaction methods ---------------------------------------------------
      /**
       *
       * These methods are supported only on `wagmi` and `ethers` since the Solana SDK does not support them in the same way.
       * These function definition is to have a type parity between the clients. Currently not in use.
       */`
I have added the implementation. lmk if it should be added or not.